Box.V2.Samples.Core.File.Upload/project.json

```diff
@@ -5,15 +5,9 @@
   },
 
   "dependencies": {
-    "Box.V2.Core": "2.*",
-    "Microsoft.NETCore.App": {
-      // include this to make it a FDB, framework depend build
-      // http://www.hanselman.com/blog/SelfcontainedNETCoreApplications.aspx
-      // dotnet build -r osx.10.10-x64
-      // dotnet publish -c release -r osx.10.10-x64
-      // "type": "platform",
-      "version": "1.0.*"
-    }
+    "Box.V2.Core": "2.14.2-*",
+    "Microsoft.NETCore.App": "1.1.0",
+    "System.Security.Cryptography.Algorithms": "4.3.0"
```

Comment:
I am adding a utility function to generate SHA. In DotNetCore, it is available in the above NuGet package. But in the next commit, I am moving it to Box.V2.Core so that this function is exposed as a Utility function. Synced with with Java and Node team and they are providing the same functions.